lighthouse-core/test/runner-test.js

```diff
@@ -155,6 +156,63 @@ describe('Runner', () => {
         expect(runAuditSpy).toHaveBeenCalled();
       });
     });
+
+    it('serializes IcuMessages in gatherMode and is able to use them in auditMode', async () => {
+      // Can use this to access shared UIStrings in i18n.js.
+      // For future changes: exact messages aren't important, just choose ones with replacements.
+      const str_ = i18n.createMessageInstanceIdFn(__filename, {});
+
+      // A gatherer that produces an IcuMessage runWarning and LighthouseError artifact.
+      class WarningAndErrorGatherer extends Gatherer {
+        afterPass(passContext) {
+          const warning = str_(i18n.UIStrings.displayValueByteSavings, {wastedBytes: 2222});
+          passContext.LighthouseRunWarnings.push(warning);
```

Comment:
lherrors have actually been serializing fine since #9397, it's other strings like runWarnings that have been causing the -G/-A problems (see https://github.com/GoogleChrome/lighthouse/issues/9269#issuecomment-512622306), so this test checks both are working.